backend/docker-compose.local.yml

```
@@ -0,0 +1,14 @@
+version: '3.8'
+
+services:
+  redis:
+    image: redis:latest
+    container_name: redis-local
+    ports:
+      - "6379:6379"
+    networks:
+      - redis_network
```

레디스도 DB이기 때문에 password를 설정해도 좋을 것 같아요.

```suggestion
    command: redis-server --requirepass password
    networks:
      - redis_network
```

만약 반영하실 경우, RedisConfig과 application.yml도 변경해주세요!